app/src/main/java/me/ccrama/redditslide/Fragments/WikiPage.java

```diff
@@ -113,10 +113,11 @@ public void onPageFinished(WebView webView, String url) {
     private void onDomRetrieved(String dom) {
         webView.loadDataWithBaseURL(
                 wikiUrl,
-                "<head>".concat(Wiki.getGlobalCustomCss())
-                        .concat(Wiki.getGlobalCustomJavaScript())
-                        .concat("</head>")
-                        .concat(dom),
+                ((("<head>"
```

Comment:
Are the triple parens required? Seems a bit odd

Reply:
Oops, nope they aren't. That was done automatically. There's other ones like this throughout the project, so I think it's fine.